src/screens/journey.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { StyleSheet, View, } from 'react-native';
+import { Text, Button } from 'react-native-paper';
+import { config } from '../config';
+import { ScreenNavigationProps } from '../routes';
+import TicketFlatList from '../components/ticketsFlatList';
+import { JourneyOutput } from '../types/JourneyOutput';
+
+const baseUrl = 'https://mobile-api-softwire2.lner.co.uk';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#eee',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type JourneyScreenProps = ScreenNavigationProps<'Journey'>;
+
+type JourneyResponse = {
+  numberOfAdults: number;
+  nextOutboundQuery: string;
+  outboundJourneys: Array<OutboundJourney>;
+};
+
+type OutboundJourney = {
+  journeyId: string;
+  departureTime: string;
+  arrivalTime: string;
+  primaryTrainOperator: TrainOperator;
+  tickets: Array<Tickets>;
+};
+
+type TrainOperator = {
+  code: string;
+  name: string;
+};
+
+type Tickets = {
+  name: string;
+  priceInPennies: number;
+};
+
+const defaultJourneyOutput: JourneyOutput = {
+  journeyID: 'Waiting for journey ...',
+  departureTime: 'Waiting for departure time ...',
+  arrivalTime: 'Waiting for arrival time ...',
+};
+
+//
+//TODO: add originStation and destinationStation functionality
+//
+const getJourney = async (
```

Comment:
It would be better to pull this method into its own file (maybe in a `src/repository/` directory) - it's not really the Journey component's 'job' to be making API requests, the component should be telling something else to make API requests